CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!--next-version-placeholder-->
 
+## v2.1.13 (2023-11-08)
```

Comment:
What's going on here? This looks identical to what is already in `main`.

Reply:
Yeah that was just me being stupid. Thought I would try out rebasing. But not really sure why I thought it would be a good idea